packages/flutter/test/widgets/sliver_fill_remaining_test.dart

```diff
@@ -495,7 +495,9 @@ void main() {
             SliverFixedExtentList(
               itemExtent: 150,
               delegate: SliverChildBuilderDelegate(
-                  (BuildContext context, int index) => Container(color: Colors.amber),
+                (BuildContext context, int index) {
```

Comment:
@Piinks - I've refactored this test to avoid checking the color(s) of the various slivers, and instead more explicitly check semantics labels corresponding to indexes.

This test would have failed because Container no longer creates a BoxDecoration in this case. Other approaches would include:

- Explicitly passing a decoration to the container we create in this test
- Using Keys - probably bad because it could affect ordering of the list, but this test doesn't seemto need that
- Refactor the test to use text widgets inside the slivers, which seemed more invasive.

Reply:
Sgtm! Thanks for updating the test!